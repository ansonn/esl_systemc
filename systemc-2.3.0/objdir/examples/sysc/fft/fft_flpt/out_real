3.016500e+04  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
-2.011000e+03  
3.217600e+04  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
2.013000e+03  
2.013721e+03  
2.013414e+03  
2.012307e+03  
2.011000e+03  
2.010127e+03  
2.010000e+03  
2.010459e+03  
2.011000e+03  
2.011108e+03  
2.010586e+03  
2.009693e+03  
2.009000e+03  
2.009045e+03  
2.010000e+03  
2.011541e+03  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
4.656000e+03  
6.138409e+02  
1.782366e+03  
1.163758e+03  
5.640000e+02  
6.947028e+02  
1.410187e+01  
3.457563e+02  
-1.200000e+02  
-4.678293e+01  
-6.783658e+02  
-2.964680e+02  
-9.720000e+02  
-1.213760e+03  
-1.598102e+03  
-4.909047e+03  
-4.536000e+03  
-4.938409e+02  
-1.662366e+03  
-1.043759e+03  
-4.440000e+02  
-5.747028e+02  
1.058981e+02  
-2.257563e+02  
2.400000e+02  
1.667829e+02  
7.983658e+02  
4.164680e+02  
1.092000e+03  
1.333760e+03  
1.718102e+03  
5.029047e+03  
5.300000e+02  
-2.477319e+03  
-9.027889e+02  
-4.555934e+02  
-2.680000e+02  
-1.969159e+01  
-1.505432e+02  
-3.459231e+02  
-2.540000e+02  
-1.449462e+02  
-4.672111e+02  
-4.058479e+02  
-3.040000e+02  
-1.220434e+02  
9.525432e+02  
2.915365e+03  
2.998000e+03  
-1.044816e+03  
6.512539e+02  
1.659765e+02  
5.280001e+02  
-3.110323e+02  
-1.566181e+02  
-6.826135e+02  
-4.420000e+02  
3.708394e+02  
-5.932539e+02  
-4.162784e+02  
4.679998e+02  
1.105009e+03  
-8.693820e+02  
3.729151e+02  
