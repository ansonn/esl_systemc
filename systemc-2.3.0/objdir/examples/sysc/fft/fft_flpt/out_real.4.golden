3.217600e+04  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
0.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
1.000000e+00  
0.000000e+00  
-1.556257e+03  
0.000000e+00  
-1.042783e+03  
0.000000e+00  
-5.724320e+02  
0.000000e+00  
-1.259072e+02  
0.000000e+00  
1.259054e+02  
0.000000e+00  
5.724309e+02  
0.000000e+00  
1.042783e+03  
0.000000e+00  
1.556260e+03  
2.112000e+03  
4.212146e+03  
-2.510606e+03  
3.096022e+02  
5.320001e+02  
-1.840003e+02  
6.365356e+01  
-3.997019e+02  
-9.080000e+02  
-4.769509e+02  
-5.193939e+02  
9.546962e+02  
-3.080001e+02  
-1.791950e+02  
2.162346e+03  
-2.892596e+03  
